_format/1.0/index.md

```diff
@@ -926,13 +926,13 @@ If an endpoint supports the `include` parameter and a client supplies it,
 the server **MUST NOT** include unrequested [resource objects] in the `included`
 section of the [compound document].
 
+If a server is unable to identify a relationship path or does not support
+inclusion of resources from a path, it **MUST** respond with 400 Bad Request.
+
```

Comment:
If the problem is with the note, then I think it's the note that should be moved up. This paragraph clearly belongs after the next one.